openfl/component/director/director.py

```diff
@@ -88,13 +88,13 @@ async def set_new_experiment(
 
         return True
 
-    def get_trained_model(self, experiment_name: str, caller: str, model_type: str):
+    def get_trained_model(self, experiment_name: str, sender: str, model_type: str):
```

Comment:
In my opinion, it is a confusing renaming. `get_trained_model` is a call, so we passed the caller's name as a parameter.
Now we pass the sender's name, and it doesn't make sense, this method will send the model back to the caller, so the sender is the Director in this case